.github/workflows/build.yml

```
@@ -0,0 +1,23 @@
+name: Build
+
+on:
+  push:
+    branches: [ main ]
+
+  pull_request:
+    branches: [ main ]
```

```suggestion
  workflow_dispatch:
  push:
  pull_request:
    branches: 
      - main
```

Add workflow dispatch to manually run the workflow 
***merging both `pull_request` and `push` to make it short***

---

Done. Thanks.